<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgMapTextPropertiesBase</class>
 <widget class="QDialog" name="DlgMapTextPropertiesBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text Properties</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin" stdset="0">
    <number>11</number>
   </property>
   <item>
    <widget class="QTabWidget" name="TextTabs">
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>&amp;Font</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin" stdset="0">
        <number>11</number>
       </property>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin" stdset="0">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin" stdset="0">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="lblFamily">
             <property name="text">
              <string>Family:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="lstFamily">
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Sunken</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin" stdset="0">
            <number>0</number>
           </property>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin" stdset="0">
            <number>0</number>
           </property>
           <item>
            <widget class="QCheckBox" name="chkBold">
             <property name="text">
              <string>Bold</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkItalic">
             <property name="text">
              <string>Italic</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="margin" stdset="0">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="lblColor_2">
               <property name="text">
                <string>Color:</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="KColorButton" name="cmdColor">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Button</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="Spacer7_2">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>S&amp;ize</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin" stdset="0">
        <number>11</number>
       </property>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin" stdset="0">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="GroupBox1">
           <property name="title">
            <string>Actual Size </string>
           </property>
           <layout class="QGridLayout">
            <property name="margin" stdset="0">
             <number>11</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="0" column="0">
             <layout class="QGridLayout">
              <property name="margin" stdset="0">
               <number>0</number>
              </property>
              <property name="spacing">
               <number>6</number>
              </property>
              <item row="0" column="1">
               <widget class="QLineEdit" name="txtWidth">
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>32767</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="Width_2">
                <property name="text">
                 <string>Width (pixels):</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="lblHeight_2">
                <property name="text">
                 <string>Height (pixels):</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="txtHeight">
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>32767</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="GroupBox2">
           <property name="title">
            <string>Font Size</string>
           </property>
           <layout class="QGridLayout">
            <property name="margin" stdset="0">
             <number>11</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="0" column="0">
             <layout class="QVBoxLayout">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="margin" stdset="0">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout">
                <property name="spacing">
                 <number>6</number>
                </property>
                <property name="margin" stdset="0">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="lblSize_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>92</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>70</width>
                    <height>32767</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Enter font size:</string>
                  </property>
                  <property name="wordWrap">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtFontSize">
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>32767</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout">
                <property name="spacing">
                 <number>6</number>
                </property>
                <property name="margin" stdset="0">
                 <number>0</number>
                </property>
                <item>
                 <spacer name="Spacer3_2">
                  <property name="orientation">
                   <enum>Qt::Orientation::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Policy::Expanding</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="cmdSetSize">
                  <property name="text">
                   <string>Set Size</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="Spacer11">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab3">
      <attribute name="title">
       <string>Text</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin" stdset="0">
        <number>11</number>
       </property>
       <item>
        <widget class="QLabel" name="lblText">
         <property name="text">
          <string>Text:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="txtText"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblPreview">
     <property name="text">
      <string>Preview:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="fraPreview">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>120</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin" stdset="0">
      <number>0</number>
     </property>
     <item>
      <spacer name="Spacer8">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdOk">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cmdCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgMapTextPropertiesBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdOk</sender>
   <signal>clicked()</signal>
   <receiver>DlgMapTextPropertiesBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkBold</sender>
   <signal>clicked()</signal>
   <receiver>DlgMapTextPropertiesBase</receiver>
   <slot>slotBoldClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkItalic</sender>
   <signal>clicked()</signal>
   <receiver>DlgMapTextPropertiesBase</receiver>
   <slot>slotItalicClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdColor</sender>
   <signal>changed(QColor)</signal>
   <receiver>DlgMapTextPropertiesBase</receiver>
   <slot>slotColorChanged(QColor)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lstFamily</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>DlgMapTextPropertiesBase</receiver>
   <slot>slotFamilySelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdSetSize</sender>
   <signal>clicked()</signal>
   <receiver>DlgMapTextPropertiesBase</receiver>
   <slot>slotSetSize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtText</sender>
   <signal>textChanged()</signal>
   <receiver>DlgMapTextPropertiesBase</receiver>
   <slot>slotUpdatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtWidth</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DlgMapTextPropertiesBase</receiver>
   <slot>slotUpdatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtHeight</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DlgMapTextPropertiesBase</receiver>
   <slot>slotUpdatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotBoldClicked()</slot>
  <slot>slotItalicClicked()</slot>
  <slot>slotColorChanged(QColor)</slot>
  <slot>slotFamilySelected()</slot>
  <slot>slotSetSize()</slot>
  <slot>slotUpdatePreview()</slot>
 </slots>
</ui>
